src/main/java/io/fabric8/maven/docker/util/Logger.java

```diff
@@ -26,13 +26,22 @@
     void info(String format, Object ... params);
 
     /**
-     * Verbose message
+     * Verbose message for build
      *
      * @param format verbose message format
      * @param params parameter for formatting message
      */
     void verbose(String format, Object ... params);
 
+    /**
+     * Verbose API messages
+     *
+     *
+     * @param format verbose message format
+     * @param params parameter for formatting message
+     */
+    void verboseApi(String format, Object ... params);
```

Comment:
Could we generalise this ? I.e. to add an an enum LogVerboseCategory with elements "Build" and "Api", and a `verbose(LogCategory cat, String format, .... params)` method ?

That would make it much easier to add more than the Api for logging.